components/mail/opendkim/Makefile

```diff
@@ -0,0 +1,104 @@
+
+COMPONENT_NAME=         opendkim
+COMPONENT_VERSION=      2.10.3
+COMPONENT_SUMMARY=      OpenDKIM is an open source implementation of the DKIM (Domain Keys Identified Mail) sender authentication system
+COMPONENT_DESCRIPTION=	OpenDKIM is an open source implementation of the DKIM \
+			(Domain Keys Identified Mail) sender authentication system proposed \
+			by the E-mail Signing Technology Group (ESTG), now standardized by \
+			the IETF (RFC6376). It also includes implementations of the RFC5617) \
+			Vouch By Reference (VBR, RFC5518) proposed standard and the experimental \
+			Authorized Third Party Signatures protocol (ATPS, RFC6541).
+COMPONENT_PROJECT_URL=  http://opendkim.org/
+COMPONENT_FMRI=         mail/opendkim
+COMPONENT_CLASSIFICATION=Applications/Internet
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=  https://sourceforge.net/projects/opendkim/files/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH= sha256:43a0ba57bf942095fe159d0748d8933c6b1dd1117caf0273fa9a0003215e681b
+COMPONENT_LICENSE=      Copyright (c) 2009, 2010, 2012, 2013, The Trusted Domain Project.
+COMPONENT_LICENSE_FILE= LICENSE
+
+include $(WS_MAKE_RULES)/common.mk
+
+PATH=			$(PATH.gnu)
+
+CONFIGURE_OPTIONS+=	--sysconfdir=/etc/mail \
+			--enable-diffheaders \
+			--enable-identity_header \
+			--enable-rbl \
+			--enable-rate_limit \
+			--enable-replace_rules \
+			--enable-atps \
+			--enable-vbr \
+			--enable-reprrd \
+			--enable-statsext \
+			--enable-stats \
+			--enable-sender_macro \
+			--with-openldap \
+			--with-openssl \
+			--with-lmdb \
+			--with-sasl \
+			--with-erlang \
+			--with-odbx \
+			--with-libcurl \
+			--with-libxml2 \
+			--with-milter \
+			--with-tre \
+			--with-libjansson \
+			--with-db \
+			--enable-sender_macro \
+			--with-libmemcached \
+			--with-lua \
+			"LIBS=-lnsl -lldap_r -lldap -llber" \
+			"OPENLDAP_LIBS=-lldap_r -lldap -llber" \
+			"LIBLUA_LIBS=" \
+			"LIBMILTER_LIBDIRS=" \
+			"CPPFLAGS=-I/usr/include/openldap -I$(SOURCE_DIR)/libut"
+
+unexport SHELLOPTS
+
+COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-64.master
+
+COMPONENT_TEST_TRANSFORMS += " -n " \
+	"-e '/TOTAL: /p' " \
+	"-e '/PASS: /p' " \
+	"-e '/SKIP: /p' " \
+	"-e '/XFAIL: /p' " \
+	"-e '/FAIL: /p' " \
+	"-e '/XPASS: /p' " \
+	"-e '/ERROR: /p' " \
+	"-e '/^=======/p' "
+
+# Auto-generated dependencies
+REQUIRED_PACKAGES += SUNWcs
+REQUIRED_PACKAGES += database/berkeleydb-5
+REQUIRED_PACKAGES += database/lmdb
+REQUIRED_PACKAGES += image/rrdtool
+REQUIRED_PACKAGES += library/jansson
+REQUIRED_PACKAGES += library/libmemcached
+REQUIRED_PACKAGES += library/libmilter
+REQUIRED_PACKAGES += library/opendbx
+REQUIRED_PACKAGES += library/openldap
+REQUIRED_PACKAGES += library/security/openssl
+REQUIRED_PACKAGES += library/tre
+REQUIRED_PACKAGES += runtime/lua
+REQUIRED_PACKAGES += runtime/perl-534
+REQUIRED_PACKAGES += shell/ksh93
```

Comment:
When I run "gmake REQUIRED_PACKAGES" on my system it'll replaces runtime/perl-534 with runtime/perl-524.

Reply:
with new setup it is the same on my system.